Mathlib/LinearAlgebra/AffineSpace/AffineEquiv.lean

```diff
@@ -561,6 +561,25 @@ theorem pointReflection_fixed_iff_of_module [Invertible (2 : k)] {x y : P₁} :
     pointReflection k x y = y ↔ y = x :=
   ((injective_pointReflection_left_of_module k y).eq_iff' (pointReflection_self k y)).trans eq_comm
 
+/-- Construct an affine equivalence from a linear equivalence and two base points.
+
+Given a linear equivalence `A : V ≃ₗ[k] V` and base points `f₀ g₀ : P`, this constructs
+the affine equivalence `T x = A (x -ᵥ f₀) +ᵥ g₀`. This is the standard way to convert
+a linear automorphism into an affine automorphism with specified base point mapping. -/
+def ofLinearEquiv {k : Type*} {V : Type*} {P : Type*}
```

Comment:
```suggestion
def ofLinearEquiv {k : V P : Type*}
```
Same below. Even better would be to have these declared in a `variable` somewhere